manifest.md

```diff
@@ -3,15 +3,15 @@
 There are three main goals of the Image Manifest Specification.
 The first goal is content-addressable images, by supporting an image model where the image's configuration can be hashed to generate a unique ID for the image and its components.
 The second goal is to allow multi-architecture images, through a "fat manifest" which references image manifests for platform-specific versions of an image.
-In OCI, this is codified in a [Manifest List](manifest-list.md).
+In OCI, this is codified in a [Image Index](image-index.md).
```

Comment:
“a Image” → “an Image” (and this “a i…” → “an i…” shows up in a few other places too).